Games/Game

```diff
@@ -0,0 +1,459 @@
+var = (int(randomNumber))
+var1 = (int(randomNumber + 1))
+
+var2 = (int(randomNumber + 3))
+var3 = (int(randomNumber + 2))
+
+var4 = (int(randomNumber + 26))
+var5 = (int(randomNumber + 22))
+
+var6 = (int(randomNumber + 3))
+var7 = (int(randomNumber + 6))
+
+var8 = (int(randomNumber * 3))
+var9 = (int(randomNumber + 37))
+
+var10 = (int(randomNumber + 112))
+var11 = (int(randomNumber + 86))
+
+var12 = (int(randomNumber + 4))
+
+var13 = (int(randomNumber * 2))
+
+var14 = (int(randomNumber + 5))
+
+var15 = (int(randomNumber + 18))
+
+var16 = (int(randomNumber + 3))
+
+var17 = (int(randomNumber + 16))
+var18 = (int(randomNumber + 14))
+
+var19 = (int(randomNumber + 17))
+var20 = (int(randomNumber + 23))
+
+var21 = (int(randomNumber + 44))
+var22 = (int(randomNumber + 44))
+
+#""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+x = 0
+score = x
+
+
+#---------------------------------------------------------------------------------------
+#this is for our calculator
+# This function adds two numbers
+def add(x, y):
+    return x + y
+
+
+# This function subtracts two numbers
+def subtract(x, y):
+    return x - y
+
+
+# This function multiplies two numbers
+def multiply(x, y):
+    return x * y
+
+
+# This function divides two numbers
+def divide(x, y):
+    return x / y
+
+
+# Import math Library
+import math
+
+import pi
+# Print the value of pi
+#print (math.pi)
+
+(pi) = round((math.pi), 3)
+
+print('Welcome to the coolest math game ever!!!')
+print("")
+print('You can enter "hint" if your stuck but it will cost you 3 points!')
+print("")
+print('To stop the game, enter "stop"')
+print("")  #these are to make space between each question
+print("")
+##############################################################################
+#stop = break
+
+#Q1 Perimeter of a square
+
+print(f'One side of a square is {var1} m what is the perimeter of the square?')
+print('please answer using a, b, c, or d.')
+
+num1 = {var1}
+num2 = 4
+
+num1 = float(var1)
+num2 = float(4)
+answer1 = (num1 * num2)
+
+answer_1 = input(
+    f"a){var1 + 5.0}\nb){var1 - 2.0}\nc){answer1}\nd){var1+20.0}\n:")
+
+#####print("The formula for the perimeter of a square is Side Length * 4")
+
+if answer_1.lower() == "c":
+    print("Correct")
+    score = score + 1
+    print(f"You have {score} point(s)")
+else:
+    print(f"Incorrect, {var1} * 4 is {answer1}")
+
+    ###################################################################################
+
+#Q2 Area of Square
+print('')
+print(f'A square has a side length of {var2} what is the area of the square')
+print(f'please answer using a, b, c, or d.')
+
+num3 = {var2}
+
+num3 = float(var2)
+answer2 = (num3 * num3)
+
+answer_1 = input(
+    f"a){var1 + 6.0}\nb){var1 + 8.0}\nc){var1 * 3.0}\nd){answer2}\n:")
+
+##print("The formula for the area of a square is Length * Width")
+
+if answer_1.lower() == "d":
```

Comment:
Your answers are always in the same option??  So the game is not meant to be played by the same person more than once?